### PR TITLE
scheduler: update hot region scheduler default config for v2

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

Issue Number: Ref #4949

### What is changed and how does it work?

- [x] For clusters upgraded from [5.2, 6.3] versions, this configuration will remain unchanged

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

Side effects

- Possible performance regression

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by nolouch — APPROVED** [review]
lgtm.
ptal @HunDunDM

**Comment by lhy1024** [comment]
I will fix test

File: server/schedulers/hot_region_config.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func initHotRegionScheduleConfig() *hotRegionSchedulerConfig {
 		DstToleranceRatio:      1.05, // Tolerate 5% difference
 		StrictPickingStore:     true,
 		EnableForTiFlash:       true,
-		RankFormulaVersion:     "", // Use default value when it is "". Depends on getRankFormulaVersionLocked.
+		RankFormulaVersion:     "v2",
```
Comment (HunDunDM) [inline server/schedulers/hot_region_config.go:76]: no need

File: server/schedulers/hot_region_config.go [hunk 2]
```diff
@@ -305,7 +305,7 @@ func (conf *hotRegionSchedulerConfig) getRankFormulaVersionLocked() string {
 	switch conf.RankFormulaVersion {
 	case "v2":
```
Comment (HunDunDM) [inline server/schedulers/hot_region_config.go:306]:
```suggestion
	case "v1":
		return "v1"
	default: // Use "v2" when it is ""
		return "v2"
```

**Review by HunDunDM — APPROVED** [review]

**Pull request merged**